concepticondata/conceptlists/Steinthal-1875-1565.tsv

```diff
@@ -0,0 +1,1566 @@
+Steinthal-1875-1565-1004	1004	Laster	Laster	vice		
+Steinthal-1875-1565-1005	1005	Sünde	Sünde	sin	2971	SIN
+Steinthal-1875-1565-1006	1006	sündigen	sündigen	to sin		
+Steinthal-1875-1565-1007	1007	Schuld	Schuld	guilt	134	FAULT
+Steinthal-1875-1565-1008	1008	Das ist nicht meine Schuld.	Das ist nicht meine Schuld.	That is not my fault.		
+Steinthal-1875-1565-1009	1009	Gewissen	Gewissen	conscience		
+Steinthal-1875-1565-1010	1010	Wahrheit	Wahrheit	truth	1638	TRUTH
+Steinthal-1875-1565-1011	1011	Wahrhaftigkeit	Wahrhaftigkeit	truthfulness		
+Steinthal-1875-1565-1012	1012	Lüge	Lüge	lie	2826	LIE (FALSEHOOD)
+Steinthal-1875-1565-1013	1013	Lügner	Lügner	liar	3470	LIAR
+Steinthal-1875-1565-1014	1014	Irrthum	Irrthum	mistake	1293	MISTAKE
+Steinthal-1875-1565-1015	1015	Gesetz	Gesetz	law	1062	LAW
+Steinthal-1875-1565-1016	1016	Sitte	Sitte	custom	1747	CUSTOM
+Steinthal-1875-1565-1017	1017	verboten	verboten	forbidden		
+Steinthal-1875-1565-1018	1018	erlaubt	erlaubt	allowed		
+Steinthal-1875-1565-1019	1019	loben	loben	to praise	3455	COMMEND
+Steinthal-1875-1565-1020	1020	tadeln	tadeln	to reprimand		
+Steinthal-1875-1565-1021	1021	spotten	spotten	to mock		
+Steinthal-1875-1565-1022	1022	anklagen	anklagen	to accuse	1814	ACCUSE
+Steinthal-1875-1565-1023	1023	strafen	strafen	to punish	3649	PUNISH
+Steinthal-1875-1565-1024	1024	lieben	lieben	to love	923	LOVE
+Steinthal-1875-1565-1025	1025	wählen	wählen	to choose	1750	CHOOSE
+Steinthal-1875-1565-1026	1026	vorziehen	vorziehen	to favour		
+Steinthal-1875-1565-1027	1027	hassen	hassen	to hate	1607	HATE
+Steinthal-1875-1565-1028	1028	dankbar	dankbar	thankful		
+Steinthal-1875-1565-1029	1029	Dank	Dank	gratitude		
+Steinthal-1875-1565-1030	1030	habgierig	habgierig	greedy	2018	GREEDY
+Steinthal-1875-1565-1031	1031	Neid	Neid	envy	40	ENVY
+Steinthal-1875-1565-1032	1032	wohlthätig	wohlthätig	benevolent		
+Steinthal-1875-1565-1033	1033	verschwenden	verschwenden	to waste	79	WASTE
+Steinthal-1875-1565-1034	1034	sparsam	sparsam	economical		
+Steinthal-1875-1565-1035	1035	geizig	geizig	stingy	1774	STINGY
+Steinthal-1875-1565-1036	1036	Grossmuth	Grossmuth	generosity		
+Steinthal-1875-1565-1037	1037	grossmüthig	grossmüthig	generous	3632	GENEROUS
+Steinthal-1875-1565-1038	1038	verzeihen	verzeihen	to forgive	1528	FORGIVE
+Steinthal-1875-1565-1039	1039	geben (Ich gebe dir.)	geben	to give	1447	GIVE
+Steinthal-1875-1565-1040	1040	geben (Ich gebe dir.)	Ich gebe dir.	I give to you.		
+Steinthal-1875-1565-1041	1041	nehmen	nehmen	to take	1749	TAKE
+Steinthal-1875-1565-1042	1042	suchen	suchen	to search	1468	LOOK FOR
+Steinthal-1875-1565-1043	1043	finden	finden	to find	719	FIND
+Steinthal-1875-1565-1044	1044	empfangen	empfangen	to receive	694	GET
+Steinthal-1875-1565-1045	1045	halten	halten	to hold	1448	HOLD
+Steinthal-1875-1565-1046	1046	festhalten	festhalten	to clutch		
+Steinthal-1875-1565-1047	1047	zurückhalten	zurückhalten	to restrain		
+Steinthal-1875-1565-1048	1048	zurückgeben	zurückgeben	to return something		
```

Comment:
GIVE BACK?

Reply:
Steinthal-1875-1565-893 'wiedergeben' could also have the same meaning, right? But it's up to a native German speaker to decide.

Reply:
The normal meaning of "wiedergeben" is "to repeat in words" or "to play back music" - it can also mean "to return something", but only very colloquially. But I don't know about the 19th century usage, and from the context it seems to be meant in that meaning. So again a case of the same meaning twice.